app/styles/module/_feeds-and-posts.scss

```diff
@@ -0,0 +1,15 @@
+.feed-posts-module {
```

Comment:
where is this class set? MW or Feeds?

Reply:
https://github.com/Wikia/mobile-wiki/blob/dev/app/modules/feeds-and-posts.js#L52